### PR TITLE
pycodestyle: add page

### LINKED ISSUE #1604: page request: pycodestyle
webpage here - https://pypi.python.org/pypi/pycodestyle/
cli documentation here - https://github.com/pycqa/pycodestyle#example-usage-and-output

Please go through the [CONTRIBUTING](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) page to understand what is the format we use for our pages.

### PR DESCRIPTION
**Pull request by andrewda**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines 

Closes #1604

### CODE REVIEW

File: pages/common/pycodestyle.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# pycodestyle
+
+> A tool to check Python code against PEP 8 style conventions.
+
+- Check the styling of a single file:
```
Comment (agnivade) [inline pages/common/pycodestyle.md]: Change styling => style. Same for below too.

File: pages/common/pycodestyle.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`pycodestyle {{file}}`
```
Comment (agnivade) [inline pages/common/pycodestyle.md]: Probably we should add `.py` extension at the end of files for extra clarity.

File: pages/common/pycodestyle.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Check the styling of a multiple files:
+
+`pycodestyle {{file1}} {{file2}} {{file3}}`
+
+- Show only the first occurrence of an error:
+
+`pycodestyle --first {{file}}`
+
+- Show the problematic code for each error:
```
Comment (agnivade) [inline pages/common/pycodestyle.md]: I think `Show the source code for each error` sounds clearer.

File: pages/common/pycodestyle.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+> A tool to check Python code against PEP 8 style conventions.
+
+- Check the style of a single file:
+
+`pycodestyle {{file.py}}`
+
+- Check the style of a multiple files:
+
+`pycodestyle {{file1.py}} {{file2.py}} {{file3}}`
```
Comment (sbrl) [inline pages/common/pycodestyle.md]: Should `file3` have the extension `.py` too?

File: pages/common/pycodestyle.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# pycodestyle
+
+> A tool to check Python code against PEP 8 style conventions.
+
+- Check the style of a single file:
+
+`pycodestyle {{file.py}}`
+
+- Check the style of a multiple files:
```
Comment (waldyrious) [inline pages/common/pycodestyle.md]: The "a" should be removed (it appears to have been copied from the example above)

**Comment by agnivade** [comment]
I take it that @sbrl's and @waldyrious' concerns are taken care of. Merging this.

**Comment by sbrl** [comment]
@agnivade Indeed! Thanks for merging :-)

**Pull request merged**